common/src/main/java/com/example/demo/ShoeApiEndpoints.java

```diff
@@ -0,0 +1,22 @@
+package com.example.demo;
+
+/**
+ * <p>
```

Comment:
Is this standard to break lines using a non-self-closing `<p>`?
Isn't `<br />` a more standard practice?

Reply:
Noted